incubator/vault/templates/deployment.yaml

```diff
@@ -95,8 +95,13 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
+          - name: VAULT_API_ADDR
+            value: "https://$(POD_IP):8200"
+        {{- if .Values.vault.dev }}
```

Comment:
Instead of using an empty if condition, how about using `if not .Values.vault.dev` instead?

Reply:
I've updated the PR.